client/src/components/related_products/RelatedProducts.jsx

```diff
@@ -1,4 +1,7 @@
 import React from 'react';
+// import axios from 'axios';
+import RelatedProduct from './RelatedProduct.jsx';
+import YourOutfit from './YourOutfit.jsx';
 
 class RelatedProducts extends React.Component {
```

Comment:
Very similar names:  RelatedProduct vs RelatedProducts. Just a be on look out - might be easier to keep track of components later on with more variety in the names between these two.